requirements/doc.txt

```diff
@@ -5,21 +5,27 @@
 #    pip-compile --output-file requirements/doc.txt requirements/base.in requirements/doc.in
 #
 alabaster==0.7.9          # via sphinx
+amqp==2.1.4               # via kombu
```

Comment:
If you use `sphinx-apidoc` to auto-generate API docs for the Python code (which projects generated from the cookiecutter do by default), it actually tries to import each file in order to retrieve docstrings and such.  This typically fails unless all the imported libraries are available, hence the virtualenv used to run Sphinx also needs all the library's core dependencies installed.